src/Codegen/Generator.hack

```diff
@@ -119,6 +119,30 @@ class Query extends BaseObject<QueryField> {
     }
 }
 
+class Mutation extends BaseObject<MutationField> {
```

Comment:
This is pretty much the same as `Query`, we could pull this out and make it more DRY